client/src/containers/Gallery/tests/Gallery-test.js

```
@@ -471,21 +471,6 @@ describe('Gallery', () => {
     });
   });
 
-  describe('handleCreateFolder()', () => {
-    let gallery = null;
-
-    beforeEach(() => {
-      props.onCreateFolder = jest.fn();
-
-      gallery = ReactTestUtils.renderIntoDocument(
-        <Gallery {...props} />
-      );
-      gallery.handleCreateFolder();
-
-      expect(props.onCreateFolder).toBeCalled();
-    });
-  });
```

This test is trying to call a method `gallery.handleCreateFolder()` that doesn't exist - I have _no_ idea why this didn't scream at us in cms 4 but my guess it is just silently fails.
There is a proper test for the behaviour I think this was intended to test in `client/src/components/GalleryToolbar/Buttons/tests/AddFolderButton-test.js`